docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.8"
+services:
+
+  scheduler:
+    image: ocurrent/ocluster-scheduler:live
+    user: "1001:1002"
+    command:
+      - --secrets-dir=/local.data/capnp-secrets
+      - --capnp-secret-key-file=/local.data/capnp-secrets/key.cap
+      - --capnp-listen-address=tcp:0.0.0.0:9000
+      - --capnp-public-address=tcp:scheduler:9000
+      - --pools=solver
+      - --state-dir=/local.data/ocluster-scheduler
+    init: true
+    ports:
+      - "9000:9000"
+    volumes:
+      - 'local.data:/local.data'
+
+  solver-worker:
+    build:
+      dockerfile: Dockerfile
+      context: .
```

Comment:
Please add a comment pointing to the live image name for this service (it can be useful to test both)
```
    # image: ocurrent/ocluster-worker:live
```